packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.tsx

```diff
@@ -28,7 +30,16 @@ export const RadioGroup = compose<RadioGroupType>({
         return null;
       }
 
-      const { label, defaultTabbableElement, isCircularNavigation, ...mergedProps } = mergeProps(radioGroup.props, final);
+      const { label, required, defaultTabbableElement, isCircularNavigation, ...mergedProps } = mergeProps(radioGroup.props, final);
+
+      const labelComponent = (
+        <React.Fragment>
+          <Slots.label>
+            <Slots.labelText key="label">{label}</Slots.labelText>
+            {!!required && <Slots.required>{typeof required === 'string' ? required : '*'}</Slots.required>}
```

Comment:
The required prop is currently only type boolean, is it meant to support strings, too? If it's only supporting boolean you don't need the "typeof" conditional

Reply:
Hm, I matched this to the web implementation but, looking at it again, web also only supports boolean for required so I'm not sure how this works. I was assuming they had this to allow the user to pass in a different indication in replacement of the '*'. Removing the conditional for now because I don't think there's a current use case for it.